include/staff/templates/task-view.tmpl.php

```diff
@@ -78,7 +78,8 @@
             'delete' => array(
                 'href' => sprintf('#tasks/%d/delete', $task->getId()),
                 'icon' => 'icon-trash',
-                'class' => 'red button',
+		'class' => 'red button',
+                'style' => 'background-color: #ffcccc; border-radius: 6px',
```

Comment:
class stays there for tasks->task but for ticket->task we use the style attribute